### PR TITLE
(Enhancement) Make MNCH Summary widgets, MCH tiles on home Dashboard configurable

### PR DESCRIPTION
**Pull request by ODORA0**
## Requirements

- [ ] This PR has a title that briefly describes the work done including the ticket number. If there is a ticket, make sure your PR title includes the ticket number in the format `OHRI-123 My PR title`.
- [ ] My work includes tests or is validated by existing tests.

## Screenshots

https://github.com/user-attachments/assets/db7e0535-eb47-446f-8602-fb8179bac12b

https://github.com/user-attachments/assets/d9ca11b0-a1e0-450e-a0ac-d029db941a48

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -19,7 +19,7 @@
     "start:hct": "openmrs develop --backend https://openmrs-dev.globalhealthapp.net --sources packages/esm-hiv-care-treatment-app",
     "start:hp": "openmrs develop --backend https://openmrs-dev.globalhealthapp.net --sources packages/esm-hiv-prevention-app",
     "start:cervical-cancer": "openmrs develop --backend https://openmrs-dev.globalhealthapp.net --sources packages/esm-cervical-cancer-app",
-    "start:pmtct": "openmrs develop --backend https://ohri-namibia-dev.globalhealthapp.net --sources 'packages/esm-ohri-pmtct-app/'",
+    "start:pmtct": "openmrs develop --backend https://ohri-namibia-demo.globalhealthapp.net --sources 'packages/esm-ohri-pmtct-app/'",
```
Comment (hadijahkyampeire) [inline package.json:22]: Do you want this to point to demo now or it was just for testing purposes?

**Review by hadijahkyampeire — APPROVED** [review]
LGTM, thanks @ODORA0 just a minor concern for the backend server changes

File: package.json (hunk 1 above)
Comment (ODORA0) [reply package.json:22]: Yes, demo is more updated and preferred

Comment (hadijahkyampeire) [reply package.json:22]: ok

**Pull request merged**